### ISSUE TITLE
Implementation of CameraCtrl and other baselines on the metric-scale pose dataset

### ISSUE THREAD

**Issue by LeeKeyu** [ZGCTroy/RealCam-I2V#3]
Hi authors, thanks for your great contributions.
I would like to ask about your implementation of some baselines on your metric-scale pose dataset.
For instance, for CameraCtrl, which was originally trained on relative-scale pose dataset, did you change the learning rate or other parameters to train on your proposed metric-scale aligned pose dataset?
Thanks a lot for your help!

**Comment by MuteApo** [comment]
I think for controllable down-stream applications, it is common for the training loss to plateau rather than decrease significantly. Therefore, qualitative visualization or metric evaluation may be more crucial indicators of model performance than the loss curve alone. The model was trained with an effective batch size of 64, achieved through gradient accumulation. This can be adjusted based on the available computational resources. For our training logs, they have already been cleared unfortunately.

**Comment by LeeKeyu** [comment]
thank you for your reply!

**Comment by Haosonn** [comment]
> > All models were trained for 50,000 steps with a fixed learning rate of 1e-4. For the ablation study on metric-scale versus relative-scale, we maintained the same number of gradient updates for a fair comparison, even though the training set sizes might differ. And we kept any other parameters the same.
> 
> [@MuteApo](https://github.com/MuteApo) Hi thanks a lot for your reply! I tried to train CameraCtrl on RealCam-Vid dataset (only using realestate10k part, including ~4w clips), but the training loss remains at about 0.2 throughout the 50k steps. So I am wondering if the metric-scale alignment introduces much differences to the dataset. I used batch size=1 and 8 GPUs. I would be grateful if you can provide the loss curves during your implementation for metric-scale and relative-scale dataset using the same method. Thanks a lot!

Hey, LeeKeyu. Have you successfully trained with 50k steps with batch_size=1 and 8 gpus? 
I tried accelerate_train.sh script and try running 10k steps for smoke test, and found that the trained controlnet checkpoint won't behave differently on different camera trajectories. I think this is the behavior by controlnet itself(training starting from zeros. I also try scaling down the official controlnet checkpoint given by the author, and found that a 0.75x scale on the official checkpoint lead to similar results, but a 0.9x scale on it works well with different camera trajs.
So as the weights of the controlnet is gradually ascending, at which steps does the model be able to distinguish between different camera trajectories?
